client/src/main/scala/skuber/api/client/package.scala

```diff
@@ -204,25 +205,25 @@ package object client {
   }
 
   def init(config: Configuration)(implicit actorSystem: ActorSystem): KubernetesClient = {
-    init(config.currentContext, LoggingConfig(), None, defaultAppConfig)
+    init(config.currentContext, LoggingConfig(), None, defaultAppConfig, None)
   }
 
   def init(appConfig: Config)(implicit actorSystem: ActorSystem): KubernetesClient = {
-    init(defaultK8sConfig.currentContext, LoggingConfig(), None, appConfig)
+    init(defaultK8sConfig.currentContext, LoggingConfig(), None, appConfig, None)
   }
 
   def init(config: Configuration, appConfig: Config)(implicit actorSystem: ActorSystem): KubernetesClient = {
-    init(config.currentContext, LoggingConfig(), None, appConfig)
+    init(config.currentContext, LoggingConfig(), None, appConfig, None)
   }
 
   def init(k8sContext: Context, logConfig: LoggingConfig, closeHook: Option[() => Unit] = None)
       (implicit actorSystem: ActorSystem): KubernetesClient = {
-    init(k8sContext, logConfig, closeHook, defaultAppConfig)
+    init(k8sContext, logConfig, closeHook, defaultAppConfig, None)
   }
 
-  def init(k8sContext: Context, logConfig: LoggingConfig, closeHook: Option[() => Unit], appConfig: Config)
+  def init(k8sContext: Context, logConfig: LoggingConfig, closeHook: Option[() => Unit], appConfig: Config, connectionPoolSettings: Option[ConnectionPoolSettings])
```

Comment:
looks great!
just add the default value here- in order to be backward compatbile
```connectionPoolSettings: Option[ConnectionPoolSettings] = None```

Reply:
I can't do that, because overloaded methods can't be mixed with default values. I added fifth overloaded method.